test/base/TestUtils.sol

```diff
@@ -0,0 +1,27 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.16;
+
+library TestUtils {
+    // String <> uint16 binary Converter Utility
+    function uint16ToBinaryString(uint16 n) public pure returns (string memory) {
```

Comment:
Was this handwritten of borrowed somewhere safe?

Reply:
Handwritten by @BenSparksCode